analyzer/codechecker_analyzer/buildlog/log_parser.py

```diff
@@ -18,6 +18,8 @@
 import tempfile
 import traceback
 
+from codechecker_analyzer.analyzers import clangsa
```

Comment:
I've put it there because there are at least 3 places where we get the clang version from the binary right now which is not the best.
The other addition to this is that the version information is parsed so it is much easier to check for major, minor version or install directory information if needed.